src/adagrams.js

```diff
@@ -1,15 +1,119 @@
+const adaGrams = {
+  A: 9,
+  B: 2,
+  C: 2,
+  D: 4,
+  E: 12,
+  F: 2,
+  G: 3,
+  H: 2,
+  I: 9,
+  J: 1,
+  K: 1,
+  L: 4,
+  M: 2,
+  N: 6,
+  O: 8,
+  P: 2,
+  Q: 1,
+  R: 6,
+  S: 4,
+  T: 6,
+  U: 4,
+  V: 2,
+  W: 2,
+  X: 1,
+  Y: 2,
+  Z: 1,
+};
+
+const scoreChart = {
```

Comment:
The other way of doing this is making a JS object where they keys are the letters and the values are the scores.  Your solution is a bit more computationally intensive (microscopically so), but your solution is more compact and easier to write. 

I also like the `adaGrams` variable to generate the pool.